### ISSUE TITLE
Dark Matter

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5909]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The issue with the trinket is that after it procs you can keep the buff and then equip your normal trinket (just like nevermelting ice crystal but for this trinket you would need to attack something in order to proc it) then equip your usual trink. It can be exploited in raids, for example use a scrapbot which can be killed and used to proc anything on it then equip your normal trinket just before the fight starts.
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. equip dark matter
2. proc it on something with melee/range attacks
3. equip your normal trinket

**Include proofs for this behaviour**
![Screenshot_4](https://user-images.githubusercontent.com/36212275/87064595-9c916780-c218-11ea-81ea-45114cf9453f.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
item ID 46038
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
